src/main/webapp/sass/_core.scss

```diff
@@ -3155,6 +3144,17 @@ div.border {
 				color: #48896B;
 			}
 		}
+		.close {
+			font-size: 0px;
```

Comment:
The things we have to do!

Reply:
I'm looking forward to deleting these hacks if we can ever manage to free ourselves from Bootstrap 3 😉